src/components/Tooltip/Tooltip.tsx

```diff
@@ -88,6 +91,8 @@ export function Tooltip({
       onMouseLeave={handleMouseLeave}
       onMouseOver={handleMouseEnterFix}
       ref={setActivator}
+      role="tooltip"
+      aria-label={accessibilityLabel}
```

Comment:
I wonder if we should be making the aria-label the read text or the content.

Reply:
That's a good question. I added the prop for increased flexibility. Occasionally people ask for larger tooltips because they put large amounts of text in it. Although not promoted it does happen, I was thinking it might be better if the label is more concise. Happy to make the change to content if you think that's more worthwhile. 

One thing to note, changing it to content wouldn't be an API change so we could always add the accessibility prop later if we felt it's needed.

Reply:
I like the idea of reading out the content

Reply:
👍  Ill make the change

Reply:
Looks like content was changed to a react node 2-3 weeks ago so we can't use content anymore. Updated the pr according & adding more gifs below